week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,8 @@
 */
 
 var startingValue = 5;
+var result = format(add((multiply(startingValue,2)),10));
```

Comment:
Great!  
No need for the extra parentheses around the call to multiply, however. and I would leave space between each argument passed to each function, so:
`format(add(multiply(startingValue, 2), 10))`
But what you have is also correct.

Reply:
Thanks. So, I have fixed it.